test/test_bootstrap.py

```diff
@@ -44,7 +44,8 @@ def test_various_imports(self):
             import_string('this is bogus!')
 
     def test_is_zipfile(self, zip_location):
-        assert not current_zipfile()
+        with mock.patch.object(sys, 'argv', [__file__]):
```

Comment:
given what you noted in the thread, I'd just assume omit this test (or make it conditional on the OS)